everest/optimizer/everest2ropt.py

```diff
@@ -1,157 +1,183 @@
 import os
-from typing import Any, Dict, List, Optional
-
-from ropt.enums import ConstraintType, PerturbationType, VariableType
-
-from everest.config import EverestConfig
-from everest.config.sampler_config import SamplerConfig
-
-
-def _parse_sampler(ever_sampler: SamplerConfig):
-    method = "default" if ever_sampler.method is None else ever_sampler.method
-    backend = "scipy" if ever_sampler.backend is None else ever_sampler.backend
-    sampler_config: Dict[str, Any] = {"method": f"{backend}/{method}"}
-    if ever_sampler.backend_options is not None:
-        sampler_config["options"] = ever_sampler.backend_options
-    if ever_sampler.shared is not None:
-        sampler_config["shared"] = ever_sampler.shared
-    return sampler_config
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, Optional, Sequence, Tuple, Union
+from typing_extensions import TypeAlias
+
+from ropt.enums import ConstraintType, PerturbationType
+
+from everest.config import (
+    ControlConfig,
+    EverestConfig,
+    SamplerConfig,
+)
+
+ControlName: TypeAlias = Union[Tuple[str, str], Tuple[str, str, int]]
+StrListDict: TypeAlias = DefaultDict[str, list]
```

Comment:
`DefaultDict[str, list]` should contain the type of elements in the list i.e `list` -> `List[Union[str,float]]`

Reply:
its more nested then just `str`, `float`, it also includes `Tuple[str, str, int]` and `VariableType'. do we include all of that?

Reply:
No, I guess it can stay as a generic list for now.